docs/management/admin/blocklist.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[blocklist]]
+[chapter]
+= Blocklist
+
+coming[8.2.0]
+
+The blocklist allows you to prevent specified applications from running if you're certain they should never run in a host environment, extending the list of processes that {endpoint-sec} considers malicious. This is especially useful to ensure that known malicious processes aren't accidentally executed by an end user. 
+
+[NOTE] 
+=====
+In addition to configuring specific entries on the **Blocklist** page, you must also ensure that the blocklist is enabled on the {endpoint-sec} integration policy in the <<malware-protection, Malware protection settings>>. This setting is enabled by default.
+
+You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].
+=====
+
+By default, a blocklist entry is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign a blocklist entry to a specific {endpoint-sec} integration policy, which blocks the process only on hosts assigned to that policy.
```

Comment:
I think this should remain mostly singular, because it's talking about configuring things on a per-entry basis (not across multiple entries). But a single blocklist entry (representing a single process) can be assigned to multiple policies, so we can make policies plural:
```suggestion
By default, a blocklist entry is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign a blocklist entry to specific {endpoint-sec} integration policies, which blocks the process only on hosts assigned to that policy.
```